deploy/build.Dockerfile

```diff
@@ -84,5 +84,12 @@ RUN arch=`dpkg --print-architecture | sed "s/arm64/aarch64/g" | sed "s/amd64/x86
     && mv mold-2.32.1-$arch-linux /home/ubuntu/mold \
     && rm mold-2.32.1-$arch-linux.tar.gz
 
+# Install sccache
+RUN  arch=`dpkg --print-architecture | sed "s/arm64/aarch64/g" | sed "s/amd64/x86_64/g"`; \
+    cd /home/ubuntu && curl -LO https://github.com/mozilla/sccache/releases/download/v0.10.0/sccache-v0.10.0-$arch-unknown-linux-musl.tar.gz \
+    && tar zxvf sccache-v0.10.0-$arch-unknown-linux-musl.tar.gz \
+    && cp sccache-v0.10.0-$arch-unknown-linux-musl/sccache /home/ubuntu/.cargo/bin \
+    && chmod +x /home/ubuntu/.cargo/bin/sccache
```

Comment:
so this wraps cargo to intercept calls?

Reply:
this is just installing the binary, the wrapping is done by setting `RUSTC_WRAPPER=sccache` env variable